src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
I know it is not a direct translation. The technical correct translation would be "värden kunde inte hittas" but that sounds strange and is not understandable for a non-tech person. 

Ping @nicholasruunu, @tobiassjosten, @rickard2. Can I have some quick input on this?